modules/administration/pages/openscap.adoc

```diff
@@ -79,25 +79,85 @@ If your system becomes damaged through the use of custom content files, you migh
 ====
 
 When you have created your content files, you need to transfer the file to the client.
-You can do this in the same way as you move any other file, using physical storage media, or across a network with [command]``ftp`` or [command]``scp``.
+You can do this in the same way as you move any other file, using physical storage media, or across a network with Salt (e. g. https://docs.saltproject.io/en/latest/ref/cli/salt-cp.html[salt-cp] or the https://docs.saltproject.io/en/latest/ref/file_server/index.html[Salt File Server]), [command]``ftp`` or [command]``scp``.
 
 We recommend that you create a package to distribute content files to clients that you are managing with {productname}.
 Packages can be signed and verified to ensure their integrity.
 For more information, see xref:administration:custom-channels.adoc[].
 
+== Find OpenSCAP profiles
+
+Different operating systems make available different OpenSCAP content files and profiles. One content file may contain more than one profile.
+
+On RPM-based operating systems, use this command to determine the location of the available SCAP files:
+
+----
+$ rpm -ql <scap-security-guide-package-name-from-table>
+----
+
+On DEB-based operating systems, use this command to determine the location of the available SCAP files:
+
+----
+$ dpkg -L <scap-security-guide-package-name-from-table>
+----
+
+When you have identified one SCAP content file that suits your needs, list what are the available profiles on the client:
+
+----
+$ oscap info ./usr/share/xml/scap/ssg/content/ssg-sle15-ds-1.2.xml
+Document type: Source Data Stream
```

Comment:
That outputs:

----

Reply:
I don't know what you mean, it looks fine here :-?

![image](https://user-images.githubusercontent.com/50680111/117144005-b86e8700-adb1-11eb-8605-85b4f02e3e4f.png)

Reply:
I wanted to separate the command from the output of the command (two blocks).  But your way is also fine.